common/src/main/java/io/novafoundation/nova/common/view/shape/ShapeProvider.kt

```diff
@@ -61,6 +61,15 @@ fun Context.getInputBackground() = getCornersStateDrawable(
     idleDrawable = getRoundedCornerDrawable(R.color.input_background)
 )
 
+fun Context.getInputBackgroundError() = getCornersStateDrawable(
+    focusedDrawable = getRoundedCornerDrawableFromColors(
+        fillColor = getColor(R.color.input_background),
+        strokeColor = getColor(R.color.button_background_reject),
+        strokeSizeInDp = 1f
+    ),
+    idleDrawable = getRoundedCornerDrawable(R.color.input_background)
```

Comment:
Are we sure we want to hide red border when view becomes unfocused? Not sure how is that supposed to be